lib/erlef/repo.ex

```diff
@@ -3,15 +3,38 @@ defmodule Erlef.Repo do
     otp_app: :erlef,
     adapter: Ecto.Adapters.Postgres
 
-  import Ecto.Query
+  @spec count(Ecto.Schema.t(), Keyword.t()) :: integer() | nil
+  def count(schema, opts \\ []) do
+    aggregate(schema, :count, opts)
+  end
+
+  def transact(fun, opts \\ []) when is_function(fun) do
```

Comment:
Any reason particular reason why we would want to use this wrapper function over `transaction/0` or `transaction/1` ?

Reply:
I added a spec, comment, and moved the function to `Erlef.Jobs` since it's used only there.